libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -195,7 +214,17 @@ void eeprom_buffered_write_byte(uint32_t pos, uint8_t value)
   */
 void eeprom_buffer_fill(void)
 {
+#if defined(DATA_EEPROM_BASE) || defined(FLASH_EEPROM_BASE)
```

Comment:
As `DATA_EEPROM_BASE `is defined if `FLASH_EEPROM_BASE `is then I gues it is not needed here.
```suggestion
#if defined(DATA_EEPROM_BASE)
```

Reply:
right